gravitee-am-ui/src/app/components/sidenav/sidenav.component.ts

```diff
@@ -76,48 +75,11 @@ export class SidenavComponent implements OnInit, OnDestroy {
     this.licenseService
       .getLicense$()
```

Comment:
If you want, you can use the method `getExpirationDate$()` to return just the expiration date.

Reply:
as we discussed: getExpirationDate$() uses 'expiresAt' instead 'expirationDate' from the backend License object. It should be unified somehow to not make a risk of a field change and break frontend.